front-end/src/app/candidate/data-candidate/data-candidate.component.ts

```diff
@@ -150,7 +149,7 @@ export class DataCandidateComponent implements OnInit {
 
     if ( this.nameButton === buttonAdd || this.nameButton === 'Change Photo' ) {
       // Image file
-      this.fileUpload = event.target.files[0];
+      this.listfile[0] = event.target.files[0];
```

Comment:
For better readability, I recommend naming this variable `listfile[0]` to something specific as `photoUrl`.

Reply:
> For better readability, I recommend naming this variable `listfile[0]` to something specific as `photoUrl`.

Modified